src/Pages/Auth/Register/index.js

```diff
@@ -53,77 +57,105 @@ function Register() {
-          />
-        </label>
-        <label>
-          {`${t("int.email")}`}
-          <Input
-            type="email"
-            onChange={(value) => setFormData({ ...formData, email: value })}
-            disabled
-            value={formData.email}
-          />
-        </label>
-        <label>
-          {`${t("int.password")}`}
-          <Input
-            onChange={(value) => setFormData({ ...formData, password: value })}
-            type="password"
-            value={formData.password}
-          />
-        </label>
-        <label>
-          {`${t("int.passwordConfirmed")}`}
-          <Input
-            onChange={(value) =>
-              setFormData({ ...formData, passwordConfirmed: value })
-            }
-            type="password"
-            value={formData.passwordConfirmed}
-          />
-        </label>
+        <div className={"u-margin-bottom u-margin-bottom-tiny"}>
+          <label className={cx("c-label c-label--primary c-label--small")}>
+            {`${t("int.firstName")}`}
+
+            <div className={cx("u-margin-bottom", inputWrapper)}>
+              <Input disabled value={formData.firstName} />
+            </div>
+          </label>
+        </div>
+        <div className={cx("u-margin-bottom u-margin-bottom-tiny")}>
+          <label className={"c-label c-label--primary c-label--small"}>
+            {`${t("int.lastName")}`}
+            <div className={cx("u-margin-bottom", inputWrapper)}>
+              <Input disabled value={formData.lastName} />
+            </div>
+          </label>
+        </div>
+        <div className={cx("u-margin-bottom u-margin-bottom-tiny")}>
+          <label className={"c-label c-label--primary c-label--small"}>
+            {`${t("int.dateOfBirth")}`}
 
-        <Button onClick={handleSubmit}>{`${t("int.submit")}`}</Button>
+            <div className={cx("u-margin-bottom", inputWrapper)}>
+              <Input
+                type="date"
+                disabled
+                //value={formData.birthDate}
+                value={convertToDate(formData.birthDate)}
+              />
+            </div>
+          </label>
+        </div>
+        <div className={cx("u-margin-bottom u-margin-bottom-tiny")}>
+          <label className={"c-label c-label--primary c-label--small"}>
+            {`${t("int.profession")}`}
+            <div className={cx("u-margin-bottom", inputWrapper)}>
+              {" "}
+              <Input disabled value={formData.profession} />
+            </div>
+          </label>
+        </div>
+        <div className={cx("u-margin-bottom u-margin-bottom-tiny")}>
+          <label className={"c-label c-label--primary c-label--small"}>
+            {`${t("int.email")}`}
+            <div className={cx("u-margin-bottom", emailInput, inputWrapper)}>
+              <Input
+                type="email"
+                onChange={(value) => setFormData({ ...formData, email: value })}
+                //value={formData.email}
```

Comment:
Why is this commented out?

Reply:
I left it there in case that the sms message would also autofill the email along with the other values

Reply:
nice thought! Let's uncomment it then